util/sycl_exceptions.h

```diff
@@ -86,6 +86,7 @@ class matcher_exception_category : public Catch::Matchers::MatcherGenericBase {
 #if SYCL_CTS_SUPPORT_HAS_EXCEPTION_CATEGORY == 0
     // There should be no compilation failures, but every CHECK_THROWS_MATCHES
     // should fail with this matcher
+    (void)other;
```

Comment:
```suggestion
    std::ignore = other;
```

Reply:
If we're aiming for beauty and setting precedence, how about [`[[maybe_unused]]`](https://en.cppreference.com/w/cpp/language/attributes/maybe_unused) on the function arg?

Reply:
The attribute works for this case too.